protocol/messenger_communities.go

```diff
@@ -2624,6 +2629,9 @@ func (m *Messenger) requestCommunityInfoFromMailserver(communityID string, shard
 		return nil, nil
 	}
 
+	// TODO: We can force to process all messages then we don't have to wait?
+	//m.ProcessAllMessages()
```

Comment:
Is this still needed?

Reply:
yep, I left it intentionally for myself not to forget when doing https://github.com/status-im/status-go/issues/4192